act/eval.py

```diff
@@ -93,15 +96,26 @@
     camera_names = POLICY_CONFIG['camera_names']
     query_frequency = POLICY_CONFIG['num_queries']
 
-        # Reset the environment
+    # Reset the environment
     obs = env.reset()
     all_actions = None
 
+    recorder = Recorder(["robot0_eye_in_hand", "frontview", "birdview"], args.environment)
+
+    def handler(arg1, arg2):
+        recorder.save_to_cloud(downsample_factor=4)
```

Comment:
Let's add an command line flag for to enable saving to the cloud and another to record or not. This will allow us to eval without always saving to the cloud

Reply:
Good idea, added